modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/display/context/WorkflowTaskDisplayContext.java

```diff
@@ -698,6 +698,11 @@ public boolean isShowExtraInfo() {
 		return _showExtraInfo;
 	}
 
+	public boolean isWorkflowTaskReadOnly() {
```

Comment:
Hey @rijema please rename the name of the method to `isReadOnly` just to follow the same pattern in [ObjectEntryDisplayContext](https://github.com/liferay/liferay-portal/blob/cb9118d51abf4e60e660543e2ea9763d128dc1d7/modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java#L491).

Thanks!

Reply:
Done!